### PR TITLE
Removes .isInterrupted property from pause

### PR DESCRIPTION
**Pull request by lukaszsobek**
Associated Issue: #4556 

### Summary of Changes

* Removes `.isInterrupted` property from pause
* Dispatches `.why` directly

### CODE REVIEW

**Comment by bomsy** [comment]
tiny flow issues
image

File: src/components/SecondaryPanes/Breakpoints.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ type Props = {
 };
 
 function isCurrentlyPausedAtBreakpoint(pause, breakpoint) {
-  if (!pause || pause.isInterrupted) {
+  if (!pause || pause.why.type === "interrupted") {
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Breakpoints.js]: lets add a `utils/pause` helper for `isPaused`

**Comment by codehag** [comment]
Do you need any help here @lukaszsobek

**Comment by codehag** [comment]
Closing by Lukas' request, work has been taken over by zacck

**Pull request closed without merging**